Frameworks/QuartzCore/CALayer.mm

```diff
@@ -1567,7 +1567,8 @@ - (CATransform3D)sublayerTransform {
 */
 - (void)setBackgroundColor:(CGColorRef)color {
     if (color != nil) {
-        priv->backgroundColor = *(reinterpret_cast<const __CGColorQuad*>(CGColorGetComponents(color)));
```

Comment:
nit: consider adding a todo here, because this assumes a 4-component color space!